engine/apps/alerts/migrations/0012_auto_20230406_1010.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.18 on 2023-04-06 10:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Keep an eye on this migration, it might take a while to finish.

Reply:
finished in a few seconds on staging. 🤞 for production.